packages/enzyme/src/Utils.js

```diff
@@ -219,23 +219,17 @@ export function AND(fns) {
   return x => fnsReversed.every(fn => fn(x));
 }
 
-export function nodeHasProperty(node, propKey, propValue) {
+export function nodeHasMatchingProperty(node, propKey, matcher) {
```

Comment:
I could probably just inline this in `matchAttributeSelector` since its only used once, and its not exported for testing like `nodeHasProperty` was

Reply:
Sounds like a good idea, the less we export the better